src/components/Editor/Tab.js

```diff
@@ -197,15 +198,13 @@ class Tab extends PureComponent<Props> {
   }
 }
 export default connect(
-  state => {
+  (state, ownProps) => {
     const selectedSource = getSelectedSource(state);
+    const { source } = ownProps;
     return {
       tabSources: getSourcesForTabs(state),
       selectedSource: selectedSource,
-      sourceMetaData: getSourceMetaData(
-        state,
-        selectedSource && selectedSource.get("id")
-      ),
+      sourceMetaData: getSourceMetaData(state, source.get("id")),
```

Comment:
This changes it from using the root url to the actual document that is displayed, right?